pkg/router/functionHandler.go

```diff
@@ -310,7 +316,15 @@ func (roundTripper *RetryingRoundTripper) RoundTrip(req *http.Request) (*http.Re
 		}
 
 		// forward the request to the function service
-		resp, err := ocRoundTripper.RoundTrip(newReq)
+		var resp *http.Response
+		if roundTripper.funcHandler.openTracingEnabled || util.IsWebsocketRequest(newReq) {
```

Comment:
Let's add a detailed comment here about the `IsWebsocketRequest` exception. So that when we plan to remove opentracing in future this would be taken care by dev modifying code.